### PR TITLE
Add Discord identifier fields to the Team table

### PR DESCRIPTION
**Pull request by D0rs4n**
- Add `discord_role_id` to the Team table
- Add `discord_channel_id` to the Team table
- Update Models accordingly

### CODE REVIEW

File: tests/routers/conftest.py [hunk 1]
```diff
@@ -37,22 +37,26 @@ def codejam() -> models.CodeJam:
...
+                discord_channel_id=2
             ),
         ]
     )
 
 
 @pytest.fixture
 async def created_codejam(
-    client: AsyncClient, codejam: models.CodeJam, session: AsyncSession
+        client: AsyncClient, codejam: models.CodeJam, session: AsyncSession
```
Comment (ChrisLovering) [inline tests/routers/conftest.py:59]: what caused the extra indent here and below? Are we running some formatter now?

**Review by ChrisLovering — APPROVED** [review]

**Review by ks129 — APPROVED** [review]
Indents are weird but otherwise OK.

**Review by ChrisLovering — CHANGES_REQUESTED** [review]

**Review by ChrisLovering — APPROVED** [review]

**Comment by ChrisLovering** [comment]
Thanks

**Comment by fisher60** [comment]
Code cov looks good now. Appropriate covered lines.

**Pull request merged**